Header header

uint32 x_size
uint32 y_size
calibration_msgs/Point2D[] points
